### PR TITLE
updated lgtm references

### PR DESCRIPTION
**Pull request by zerotyr**
updated LGTM references to point to GitHub code scanning rather than LGTM. added links to documentation

### CODE REVIEW

**Comment by kingthorin** [comment]
They may be intertwined but results from LGTM are very different from just GitHub’s CodeQL.

At least as far as Java code analysis goes. I can’t speak to other langs.

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 1]
```diff
@@ -56,11 +56,12 @@ list of those that are "Open Source or Free Tools Of This Type". Any
...
-    source static analysis service that automatically monitors commits
-    to publicly accessible code in: Bitbucket Cloud, GitHub, or GitLab.
-    Supports C/C++, C\#, COBOL (in beta), Java, JavaScript/TypeScript,
-    Python
+  - [GitHub code scanning](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning) - A free for open
+    source static analysis service that uses [GitHub Actions](https://docs.github.com/en/actions) and [CodeQL](https://codeql.github.com/)
+    to scan public repositories on GitHub.
+    Supports C/C++, C\#, Ruby (beta), Java, JavaScript/TypeScript,
+    Python, and Go (see [here](https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/) for more information)
+    - If you do not want to use GitHub Actions, you may use the [CodeQL CLI](https://github.com/github/codeql-cli-binaries); however, be sure to read the license terms in full
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:64]:
```suggestion
    - If you do not want to use GitHub Actions, you may use the [CodeQL CLI](https://github.com/github/codeql-cli-binaries); however, be sure to read the license terms in full.
```

**Review by kingthorin — APPROVED** [review]
Thanks. I'm going to accept as-is. If someone from GitHub or LGTM want to tweak it further we can watch for other PRs :grinning:

**Comment by zerotyr** [comment]
not a problem. thanks for the quick response. my understanding is LGTM and GitHub code scanning should have feature parity. however, by default, CodeQL is configured to only look for high fidelity security results. that is, to minimize the number of false positives and only attempts to show true positives. 

LGTM may be showing extra/different results due to a different config. to get the same or similar outputs in CodeQL, I would suggest changing the default [CodeQL pack](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#using-queries-in-ql-packs) that is used to be the `security-and-quality` query pack.

if you like, I can add this in to clarify how to achieve feature parity, or at least close to it.

also, apologies, but I forgot to mention, I am from GitHub 😸

**Comment by kingthorin** [comment]
> if you like, I can add this in to clarify how to achieve feature parity, or at least close to it.

That'd probably be a great move!

> also, apologies, but I forgot to mention, I am from GitHub 😸

No worries, thanks for the contribution(s) 👍

**Pull request merged**